docs/Documentation/Development/Languages/Julia/julia_tour.md

```diff
@@ -0,0 +1,1178 @@
+---
+title: Tour of Julia
+postdate: November 9, 2022
+layout: default
+author: Jonathan Maack
+description: A brief introduction to Julia
+parent: Julia
+grand_parent: Languages
+---
+
+# Tour of Julia
+
+*"Julia aims to create an unprecedented combination of ease-of-use, power, and efficiency in a single language." --Julia Documentation*
+
+## Why Julia?
+
+Feature Highlights:
+
+* Designed for scientific computing
+* Non-vectorized code is just as fast as vectorized code
+* Designed for distributed and parallel computing
+* Call C/FORTRAN functions directly
+* Metaprogramming
+
+## Basics
+
+* [REPL (Read-Evaluate-Print-Loop)](#repl-read-evaluate-print-loop)
```

Comment:
I don't think the layouts are necessary since we have the page table of contents on the right of the page. Same for the rest of the page and the other pages.